src/components/Home/Home.js

```diff
@@ -103,7 +103,18 @@ class Home extends React.Component {
                         fontSize: lerp(showAppsProgress, 37, 22) + 'px',
                       }}
                     >
-                      Welcome to Aragon 0.5 − The Architect
+                      Welcome to Aragon!
+                    </Text>
+                    <br />
+                    <br />
```

Comment:
Generally we don't use `<br />` for just styling purposes; it would be better to directly add a `margin-top` to the following `<Text />` element.

Reply:
Fixed!